var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -134,6 +141,18 @@ class Gcc(AutotoolsPackage, GNUMirrorPackage):
              destination='newlibsource',
              when='+nvptx',
              fetch_options=timeout)
+    resource(name='newlib-amdgcn',
+             url='ftp://sourceware.org/pub/newlib/newlib-3.3.0.tar.gz',
+             sha256='58dd9e3eaedf519360d92d84205c3deef0b3fc286685d1c562e245914ef72c66',
+             destination='newlibsource',
+             when='+amdgcn',  # TODO: Is there an or-selection syntax?
+             fetch_options=timeout)  # to combine with newlib, above
```

Comment:
What will happen if both nvptx and amdgcn are enabled? They seem to extract into the same destination.